cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,298 @@
+/// <reference types="cypress" />
+
+context('Actions', () => {
```

Comment:
Maybe I missed your reply on my feedback during the previous review, but why do we add all these sample tests into our project?

Reply:
I've added them as a reference, but could also delete them.

Reply:
I'd advise to delete them as they serve no purpose.